discovery-provider/requirements.txt

```diff
@@ -12,6 +12,7 @@ types-pytz==2021.1.2
 # web3 used to be 5.11.0 but due to a build issue in docker, downgraded to 5.8.0
 web3==5.8.0
 Flask==1.0.4
+itsdangerous==2.0.1
```

Comment:
i think this is on master?

Reply:
Not yet unfortunately: https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/requirements.txt

Reply:
oh got it this is going to be incoming once we reconcile current release - https://github.com/AudiusProject/audius-protocol/pull/2570/commits/b806f74ccb9d5db72ece6348baa1fa4aa75965f3

Reply:
resolving this too